x-pack/plugins/ml/public/application/datavisualizer/file_based/components/file_datavisualizer_view/file_error_callouts.tsx

```diff
@@ -81,7 +80,7 @@ interface FileCouldNotBeReadProps {
 }
 
 export const FileCouldNotBeRead: FC<FileCouldNotBeReadProps> = ({ error, loaded }) => {
-  const message = error.body.message;
+  const message = error?.body?.message || '';
```

Comment:
Could we make use of `getErrorMessage()` from `plugins/ml/common/util/errors.ts` here?